e3sm_to_cmip/lib.py

```diff
@@ -735,8 +735,14 @@ def load_axis(data, levels=None, has_time=True):
 
         axis_bnds = levels.get("e3sm_axis_bnds")
         if axis_bnds:
+            cell_bounds = data[axis_bnds]
+            # i.g. handler clcalipso returns xarray dataarray
```

Comment:
```suggestion
            # i.g. handler cl, cli, clw returns xarray dataarray
```

> Looping over all atm variables, fix in commit https://github.com/E3SM-Project/e3sm_to_cmip/commit/4f7a0a45bd8617d9350d48ca192cdfe400c76b90 resulted "clw, cl, cli failed to complete".

I think `clcalipso` does return a numpy array since the `write_data()` function returns `.values`.

https://github.com/E3SM-Project/e3sm_to_cmip/blob/4dcf50676733d7ed3123a6efca7d9f9e1b3170e3/e3sm_to_cmip/cmor_handlers/clcalipso.py#L23-L33

The error messages points to `clw`, `cl`, and `cli` handlers not returning `.values` in their `write_data()` functions:

https://github.com/E3SM-Project/e3sm_to_cmip/blob/4dcf50676733d7ed3123a6efca7d9f9e1b3170e3/e3sm_to_cmip/cmor_handlers/cli.py#L24
https://github.com/E3SM-Project/e3sm_to_cmip/blob/4dcf50676733d7ed3123a6efca7d9f9e1b3170e3/e3sm_to_cmip/cmor_handlers/clw.py#L26
https://github.com/E3SM-Project/e3sm_to_cmip/blob/4dcf50676733d7ed3123a6efca7d9f9e1b3170e3/e3sm_to_cmip/cmor_handlers/cl.py#L26

---

Your implementation is much more efficient than repeatedly calling `.values` in each of the handlers. We won't have to worry about this anymore in #140.